webapp/packages/core-ui/src/Form/FormState.ts

```diff
@@ -21,27 +21,30 @@ import { formStateContext } from './formStateContext';
 import type { IFormPart } from './IFormPart';
 import type { IFormState } from './IFormState';
 
+type PartsValues = IFormPart<any>[];
```

Comment:
this type is too simple, probably better to inline it, cuz usually you don't declare something like this:
```
type Strings = string[]
```